http/src/response/mod.rs

```diff
@@ -112,10 +112,12 @@ impl Display for DeserializeBodyError {
             &DeserializeBodyErrorType::BodyNotUtf8 { .. } => {
                 f.write_str("response body is not a utf-8 valid string")
             }
-            DeserializeBodyErrorType::Chunking { .. } => {
-                f.write_str("failed to chunk response body")
+            DeserializeBodyErrorType::Chunking => f.write_str("failed to chunk response body"),
+            #[cfg(feature = "compression")]
+            DeserializeBodyErrorType::Decompressing => {
```

Comment:
```suggestion
            DeserializeBodyErrorType::Decompressing { .. } => {
```
Let's add this so if we ever add a field to this variant, it's less breaking.